backend/src/controllers/reviewController.ts

```diff
@@ -23,6 +25,37 @@ export const addReview = async (req: Request, res: Response) => {
   }
 }
 
+export const deleteReview = async (req: Request, res: Response) => {
+  const jwt = req.cookies.authToken;
+  if (!jwt) {
+    res.status(400).json({ message: 'Missing JWT cookie' });
+  }
+  let id: number;
+  try {
+    const payload = auth.verifyAndReadJWT(jwt);
+    id = payload.id;
```

Comment:
technically its bad to have this ID outside if we're not using it there. where you pull it, just do "const id = payload.id"